omod/src/main/webapp/datasets/datasetEditor.jsp

```diff
@@ -38,16 +37,46 @@
 				$j('#fixedValue${p.field.name}').show();
 			}
 		</c:forEach>
-
+		
 	} );
+	
+	$j(document).ready(function() {
+		$j('#loadAffectedReports').click(function() {
+			$j.ajax({
+				type: 'POST',
+			    contentType : "application/json",
+			    url: 'loadAffectedDatasetDefs.form?uuid=' + '${dataSetDefinition.uuid}',
+			    dataType: 'json',
+			    success: function(json) {
+			    	$j('#affectedReports').html('');
+				    $j('#affectedReports').append('<ul></ul>')
+		                if (json.length > 0) {
+		            	    for (i in json) {
+		        		var uuid = uuid = json[i].uuid;
+		                        var url = "${pageContext.request.contextPath}/module/reporting/reports/reportEditor.form?uuid=" + uuid;
+		        		$j('#affectedReports ul').append('<li><a href=' + url + '>' + json[i].name + "</a></li>");	        	 
+		                    }
 
+		                } else {
+		        	    $j('#affectedReports').append('<li><b>No Affected Report Defifnitions!</b></li>')
+		                }
+			    },
+			    error: function() {
+				    alert("An error has has occured!");
+			    }
```

Comment:
Rather than use an alert box, why not display an error message in the same area that the affected report definitions would go.  eg. where you say "No Affected Report Definitions" if none are returned, you would show "Error retrieving affected report definitions" if an error occurs.  Also log the error to the console for easier debugging.  You'll also want to localize this error message.